README.md

```diff
@@ -3,11 +3,10 @@
 The Kubernetes Template Project is a template for starting new projects on the github.com/kubernetes organization. All Kubernetes projects, at mimimum, must have the following files:
```

Comment:
The document notes it's for things on the github.com/kubernetes organization. But, it's being proposed for uses outside of that (other orgs). Should that be updated?

Reply:
I would like to change the wording to be something like "new projects in a kubernetes github organization" with a link to a list of those organizations.  But we don't yet have that list merged.  I was planning to do so in a follow on PR.